cmd/ddev/cmd/start.go

```diff
@@ -137,7 +137,7 @@ ddev start --all`,
 
 			util.Success("Successfully started %s", project.GetName())
 			httpURLs, httpsURLs, _ := project.GetAllURLs()
-			if !nodeps.IsGitpod() && !nodeps.IsCodespaces() && (globalconfig.GetCAROOT() == "" || ddevapp.IsRouterDisabled(project)) {
+			if !nodeps.IsGitpod() && !nodeps.IsCodespaces() && (globalconfig.GetCAROOT() == "" || ddevapp.IsRouterDisabled(project)) && !project.IsCustomCert() {
```

Comment:
This needs to remove `"github.com/ddev/ddev/pkg/nodeps"` at the top of this file

```suggestion
			if project.CanUseHTTPOnly() {
```